spaces/aliases.json

```
@@ -65,5 +65,6 @@
   "arbitrum-odyssey.eth": "arbitrumfoundation.eth",
   "pokt-network": "poktdao.eth",
   "lensterapp.eth": "lenster.xyz",
-  "nexondao.eth": "eralend.eth"
+  "nexondao.eth": "eralend.eth",
+  "nedao.eth": "nedao.eth"
```

It looks like you don't need to migrate because you've used the same 2 ENS.
```suggestion
  "nedao.eth": "nedao.eth"
```